components/adblock_rust_ffi/Cargo.toml

```diff
@@ -5,7 +5,8 @@ authors = ["Brian R. Bondy <netzen@gmail.com>"]
 edition = "2018"
 
 [dependencies]
-adblock = { version = "0.6.3", default-features = false, features = ["full-regex-handling", "object-pooling", "unsync-regex-caching"] }
+# TODO: Revert
```

Comment:
Will be reverted after https://github.com/brave/brave-browser/issues/27967 (before merging this PR)

Reply:
we could also include the version bump as part of this PR, if that's easier

Reply:
(published [`0.7.0`](https://crates.io/crates/adblock/0.7.0))

Reply:
Thanks, updated.